cfgov/jinja2/v1/_includes/organisms/snippet-list.html

```diff
@@ -37,7 +37,7 @@
 
 {% from 'molecules/info-unit.html' import info_unit with context %}
 
-<div class="o-snippet-list" id="{{ value.heading|slugify }}">
+<div class="o-snippet-list" {{ 'id="' ~ value.heading|slugify ~ '"' if value.heading else '' }}>
```

Comment:
If I really want to nitpick, you could do `…list"{{ ' id=… ` to prevent outputting that space when the condition is false :)